pkg/controllers/disruption/helpers.go

```diff
@@ -106,6 +107,10 @@ func SimulateScheduling(ctx context.Context, kubeClient client.Client, cluster *
 				if _, ok := deletingNodePodKeys[client.ObjectKeyFromObject(p)]; !ok {
 					results.PodErrors[p] = NewUninitializedNodeError(n)
 				}
+				// check for a pod that was not scheduled due to ctx timeout
```

Comment:
I'm probably missing something, but what is this for? Shouldn't we normally fail scheduling if there is a pod error?

Reply:
This is probably unnecessary. Was using it to error on line 142 after calling `SimulateScheduling` in `computeConsolidation` but we do check if pods failed to schedule on line 146. We'd then return empty results, no command nil error, and continue the loop. 

The loop then either breaks because min > max and returns the last saved items or the timeout bails out of consolidation and returns the last saved items plus an error if context deadline was exceeded.